lib/binding.ex

```diff
@@ -4,11 +4,13 @@ defmodule GenRMQ.Binding do
   declaring consumer bindings and exchanges.
   """
 
-  @type exchange :: String.t() | {exchange_kind(), String.t()}
+  @type exchange :: String.t() | {exchange_kind(), String.t()} | :default
```

Comment:
Can we add something that explains `:default` in the documentation here as well? I think it would be the first place I would look.

Reply:
@vorce good point. Will update / extend the documentation.

Reply:
@vorce added some docs. Let me know if this is smth you had in mind.

Reply:
@mkorszun yes, perfect 👍